package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "dependencies": {
     "chalk": "~0.4.0",
-    "lodash": "^3.10.1",
+    "lodash": "^4.5.0",
```

Comment:
Bumping lodash is fine, as long as you check the breaking changes and do it in a separate commit. We want to make sure we keep separate concerns. (`chore(npm): lodash ^4.5.0`, then the rest)